Objective-C/CBLReplicatorConfiguration.m

```diff
@@ -215,6 +247,12 @@ - (NSDictionary*) effectiveOptions {
     if (_heartbeat > 0)
         options[@kC4ReplicatorHeartbeatInterval] = @(_heartbeat);
     
+    if (_maxRetryWaitTime > 0)
+        options[@kC4ReplicatorOptionMaxRetryInterval] = @(_maxRetryWaitTime);
+    
+    if (_maxRetries >= 0)
+        options[@kC4ReplicatorOptionMaxRetries] = @([self maxRetries]);
```

Comment:
`if (_maxRetries >= 0)` should be removed?

Reply:
:(